manifests/u/ULAB/PaintAid/2.3.4.1/ULAB.PaintAid.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=QUSU.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ULAB.PaintAid
+PackageVersion: 2.3.4.1
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- hhzs
+FileExtensions:
+- mtc
+- mtd
+- mtp
+ReleaseDate: 2023-02-16
+Installers:
+- Architecture: x86
+  InstallerUrl: https://s.udongman.cn/paintaid/32/HHZS.2.3.4.1.20230216.win.exe
+  InstallerSha256: 901DEE2C9F5574FFBF8546BC8EBA68F17702A309DC964BA9A493D9A3DCC5F546
```

Hi @SpecterShell, this is the correct hash for this installer.
```suggestion
  InstallerSha256: E351BF44E556AB72DFC91FE19111BA8064912EB83F7EE36CF803A6339FC410BB
```